### PR TITLE
feat(hpc-vmware-managed-vcd): add general_info quick_actions

### PR DESCRIPTION
**Pull request by pauldkn**
ref: MANAGER-14526

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/hpc-vcd-beta`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes
| Tickets          | Feat #MANAGER-14526
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-general-information-tile/OrganizationGeneralInformationTile.component.tsx [hunk 1]
```diff
@@ -48,14 +52,42 @@ export default function OrganizationGenerationInformationTile({
...
+          onEdit={() => {}}
+        />
+      )}
+      {showModal === 'description' && (
+        <EditDescriptionModal
+          organizationDescription={vcdOrganization?.currentState?.fullName}
+          onCloseModal={() => setShowModal(false)}
+          onEdit={() => {}}
+        />
+      )}
```
Comment (chipp972) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-general-information-tile/OrganizationGeneralInformationTile.component.tsx:72]: Modals have their own route and urls :
https://github.com/ovh/manager/blob/master/packages/manager/apps/pci-rancher/src/routes.tsx

Take example on the pci-rancher project with the EditModal or GenerateAccessModal

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDescriptionModal.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      onOdsModalClose={onCloseModal}
+      dismissible
+    >
+      <OsdsText
+        color={ODS_THEME_COLOR_INTENT.text}
+        level={ODS_TEXT_LEVEL.heading}
+        size={ODS_TEXT_SIZE._400}
+      >
+        {t('managed_vcd_dashboard_edit_desc_modal_title')}
+      </OsdsText>
```
Comment (rjamet-ovh) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDescriptionModal.tsx:57]: you can use the `OsdsModal` `headline` prop for the title

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDescriptionModal.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+      <OsdsFormField>
+        <OsdsText color={ODS_THEME_COLOR_INTENT.text} className="mt-6">
+          {t('managed_vcd_dashboard_edit_desc_modal_label')}
+        </OsdsText>
```
Comment (rjamet-ovh) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDescriptionModal.tsx:62]: you should be able to use the `label` slot
`<osds-text slot='label' color='text'>Label</osds-text>`

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditNameModal.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+          color={
+            isValidName
+              ? ODS_THEME_COLOR_INTENT.info
+              : ODS_THEME_COLOR_INTENT.error
+          }
+        />
+        <OsdsText
+          slot="helper"
+          color={ODS_THEME_COLOR_INTENT.error}
+          className={`${isValidName ? 'invisible' : 'visible'}`}
```
Comment (vovh) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditNameModal.tsx:74]: className={isValidName ? 'invisible' : 'visible'}

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDescriptionModal.tsx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+          color={
+            isValidDesc
+              ? ODS_THEME_COLOR_INTENT.info
+              : ODS_THEME_COLOR_INTENT.error
+          }
+        />
+        <OsdsText
+          slot="helper"
+          color={ODS_THEME_COLOR_INTENT.error}
+          className={`${isValidDesc ? 'invisible' : 'visible'}`}
```
Comment (vovh) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDescriptionModal.tsx:74]: className={isValidDesc ? 'invisible' : 'visible'}

**Review by vovh — CHANGES_REQUESTED** [review]

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDescriptionModal.tsx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+}: EditModalProps) => {
+  const { t } = useTranslation('dashboard');
+  const [newDesc, setNewDesc] = useState<string>(organizationDescription || '');
+  const isValidDesc = validateDescription(newDesc);
+  const isButtonEnabled = isValidDesc && newDesc !== organizationDescription;
+
+  const handleSubmit = () => {
+    if (isValidDesc) {
+      onEdit(newDesc);
+      onCloseModal();
```
Comment (chipp972) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDescriptionModal.tsx:39]: We should close the modal on success of the onEdit mutation

**Review by chipp972 — APPROVED** [review]

**Review by vovh — APPROVED** [review]

**Pull request merged**